### DISCUSSION TITLE
sendVideo 'caption ' - "Unknown named parameter $caption"

### DISCUSSION THREAD

**Discussion by FeistySquirrel** [nutgram/nutgram#650] [Q&A]
I've written many bots with NodeJS, but chose Nutgram for my first PHP bot endeavour.

Any pointers as to why the 'caption' parameter on sendVideo isn't recognised?  Coming back with "Unknown named parameter".

`	 $bot->sendVideo(
		video: 'https://VideoUrl.com',
        caption: 'my caption',
        parse_mode: ParseMode::HTML,
		  );`

**Comment by Lukasss93** [comment]
Can you please open an issue?

**Reply by FeistySquirrel** [reply]
Hi, Sure thing.  I'll do it now.  I wasn't sure if it was  an issue or a user error.
